### PR TITLE
look for benchview files in bin/debug/tools

### PR DESCRIPTION
**Pull request by TyOverby**

### CODE REVIEW

**Comment by TyOverby** [comment]
@dotnet-bot test perf please

**Comment by TyOverby** [comment]
@dotnet-bot test perf please

**Comment by TyOverby** [comment]
@dotnet-bot test perf please

**Comment by TyOverby** [comment]
@jaredpar I think this broke when you made a bunch of nested subfolders in the bin/release directory.

**Comment by TyOverby** [comment]
@dotnet/roslyn-infrastructure @dotnet/roslyn-perf: Please review.  Very small change.

**Comment by TyOverby** [comment]
@dotnet-bot test perf please

File: cibuild.cmd [hunk 1]
```diff
@@ -102,6 +102,7 @@ if defined TestPerfRun (
 
             REM Get the benchview tools - Place alongside Roslyn.Test.Performance.Runner.exe
             call "%RoslynRoot%\build\scripts\install_benchview_tools.cmd" ".\Binaries\%BuildConfiguration%\" || goto :BuildFailed
+            dir ".\Binaries\%BuildConfiguration%\"
```
Comment (jaredpar) [inline cibuild.cmd]: Why do we need this?

Comment (TyOverby) [inline cibuild.cmd]: I'm still debugging this... Apparently if you try to do a `nuget install` into a folder that doesn't exist, nuget will say `Unable to find package 'Microsoft.BenchView.JSONFormat'`.

**Comment by TyOverby** [comment]
@dotnet-bot test perf please

**Comment by TyOverby** [comment]
@dotnet-bot test perf please

**Pull request merged**